.env.example

```diff
@@ -1,21 +1,22 @@
+CONTAINER_PREFIX="rapidez_"
+# Used by docker-compose to define your published domains
```

Comment:
Maybe move the docker specific settings which won't be used with a normal installation to the bottom?